tests/scr-readingsystem-features/OPS/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Scripting</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-12-10</dc:date>
+    <dc:description>Checks whether the reading system implements the required epubReadingSystem feature string queries.</dc:description>
+    <dc:identifier id="pub-id">scr-readingsystem-features</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>epubReadingSystem object features</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#scripting-req-readingsystem</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#scripting-req-readingsystem-features</meta>
```

Comment:
These two IDs don't exist. Is that because you're adding them in a follow PR? Fine if so.

Reply:
Yep, they are in https://github.com/w3c/epub-specs/pull/1956